server/protocols/jmap-rfc-8621-integration-tests/distributed-jmap-rfc-8621-integration-tests/pom.xml

```diff
@@ -17,19 +17,17 @@
     specific language governing permissions and limitations
     under the License.
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
it's not mandatory but because whitespace is considered non-significant in the XML specification parsers don't keep position information in many cases this is one of them. Since
- there were few pom affected by the whitespace removal 
- this will prevent spurious modifications that humans will have to think about
- the namespace declaration on the project tag are more or less noise to humans
I added the few normalizations to the PR

Reply:
no problem as far as it is not mandatory